core/src/main/java/hudson/model/DirectoryBrowserSupport.java

```diff
@@ -446,6 +455,29 @@ public long getSize() {
             return size;
         }
 
+        /**
+         *
+         * @return A long value representing the time the file was last modified, measured in milliseconds since
+         * the epoch (00:00:00 GMT, January 1, 1970), or 0L if is not possible to obtain the times.
+         * @since TODO
+         */
+        public long getLastModified() {
+            return lastModified;
+        }
+
+        /**
+         *
+         * @return A Calendar representing the time the file was last modified, it lastModified is 0L
+         * it will return 00:00:00 GMT, January 1, 1970.
```

Comment:
Can we agree that 0 being 1970 is known to never be the correct answer, and we should instead not show anything in that case?

Reply:
condition added in the jelly